### DISCUSSION TITLE
Versioning idea

### DISCUSSION THREAD

**Discussion by cheekoo** [tteck/Proxmox#337] [Ideas]
First off, thanks for putting these together. Very helpful. I frequently use a couple of them. You use versioning system as in *-v2 or *-v3. With that users can not have 'hard links' to your scripts. Do you think we could just keep the name of most recent the same, so we can just clone your repo on proxmox machine, update and run it without coming to the github page?
Just an idea. thanks again!

**Comment by fbartels** [comment]
> so we can just clone your repo on proxmox machine

I think this would not really work either. I did look a bit into the scripts and since they download further scripts at runtime (for example at https://github.com/tteck/Proxmox/blob/dc4da55fc366187b40153bcf1bf2e15473f4e16b/ct/docker-v3.sh#L340) you cannot "freeze" your current version.

**Reply by mschabhuettl** [reply]
You can cross link it. For example: grafana-latest.sh is downloading gafana-v4.sh. Just an idea!

**Comment by tteck** [comment]
FYI, the version has nothing to do with the program it's installing. It is about the installation method. For example v2 and v3 use the same -install.sh, v3 has many more user options to export over than v2 when creating the LXC. You can still use any version, just change v3 to v2 or v1 
example:
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/debian-v2.sh)"
```
v1 of the script uses a different -install.sh called _setup.sh
